concent_api/core/models.py

```diff
@@ -604,9 +651,11 @@ def response_type_enum(self) -> 'ResponseType':
     created_at = DateTimeField(auto_now_add=True)
     modified_at = DateTimeField(auto_now=True)
 
+    payment_info = OneToOneField(PaymentInfo, related_name='payments', blank=True, null=True)
```

Comment:
Related name should be `payment_info`, `payments` suggests a collection.

Reply:
Is it even necessary to explicitly set `related_name` here?

Reply:
BTW, don't forget to regenerate the migration if you change it.

Reply:
`related_name` is not necessary, removed. I remembered about migration, but thanks :)